fix: FIT-248: Adapt scrollbars to dark mode

carly-bartel opened the pull request:
### Reason for change

Scrollbars on several pages in Dark Mode were appearing white, which was a regression as per Ricardo Cabral. This was due to some elements missing the `styled-scrollbar` mixin, preventing them from adopting the correct dark mode styling.

### Screenshots

**Before**
<img width="1092" height="611" alt="Screenshot 2025-07-15 at 1 34 46 PM" src="https://github.com/user-attachments/assets/916d26a5-0741-4bbd-b7e7-9eecff86d2a7" />
<img width="1090" height="613" alt="Screenshot 2025-07-15 at 1 35 13 PM" src="https://github.com/user-attachments/assets/28384b9f-a0e9-47a8-9ba2-890e44052936" />
<img width="1087" height="611" alt="Screenshot 2025-07-15 at 1 35 25 PM" src="https://github.com/user-attachments/assets/f139affe-4626-4f15-b7c6-52e5bae6adb9" />


**After**
<img width="896" height="438" alt="Screenshot 2025-07-15 at 1 38 08 PM" src="https://github.com/user-attachments/assets/2f198ef8-bbcd-45e4-8c2a-80f42753a525" />
<img width="1096" height="578" alt="Screenshot 2025-07-15 at 1 38 30 PM" src="https://github.com/user-attachments/assets/51689089-cbbc-474e-b5ab-5ee89570a251" />
<img width="1405" height="784" alt="Screenshot 2025-07-15 at 1 38 42 PM" src="https://github.com/user-attachments/assets/9c8aa0d1-25f7-45cc-801b-8b8c31fe6a12" />


### Testing

The fix was verified by following the steps to reproduce the original bug:

1.  As OWNER, log in to `https://ls-release.appx.humansignal.com/`
2.  Click on Create project button
3.  Observe that Create Project modal pops up
4.  Navigate to data import tab
5.  Import provided data: `TSSyncData.json`
6.  Navigate to Labeling Interface tab
7.  Click on Custom template option
8.  Add provided config: `AudioVideoTSSyncConfig.txt`
9.  Click on Save button
10. Observe that you’re at Data Manager
11. Click on imported task
12. Observe that you’re at Quick View
13. Confirm that several scrollbars are now adapted to Dark Mode.
14. Navigate to Settings > Labeling Interface page
15. Confirm that the horizontal scrollbar is now adapted to Dark Mode.
16. Navigate to Organization page
17. Confirm that the scrollbar on the right side is now adapted to Dark Mode.

### Reviewer notes

The changes primarily involve removing the 'styled-scrollbar' mixin and applying them globally in the tailwind.css file

carly-bartel committed:
fix: FIT-248: Scrollbars are white on several pages in Dark mode

carly-bartel committed:
add styled-scrollbar mixin to sidebarmenu, table and sidepanels

carly-bartel committed:
fixed sidebar-menu content so that there was no horizontal overflow scrolling on labeling interface

yyassi-heartex commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16319552989)

yyassi-heartex committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16319552989

hlomzik reviewed (changes requested):
Can we move this mixin to a dedicated file and import it? That would make it universal and easily accessible in any other places

carly-bartel committed:
remove custom scrollbar mixins and replaced with global scrollbar style

carly-bartel committed:
Merge branch 'fb-fit-248/scrollbars-white' of https://github.com/HumanSignal/label-studio into fb-fit-248/scrollbars-white

carly-bartel commented:
> Can we move this mixin to a dedicated file and import it? That would make it universal and easily accessible in any other places

@hlomzik i removed all of the mixins and just applied the scrollbar style globally in the tailwind.css file

carly-bartel committed:
lint fixes

yyassi-heartex commented on web/libs/editor/src/assets/styles/_mixins.scss:
```diff
@@ -1,30 +1,15 @@
 @mixin respond($breakpoint) {
-  @if $breakpoint == "phone" {
-    @media (width <= 760px) {
+  @if $breakpoint =="phone" {
```
```suggestion
  @if $breakpoint == "phone" {
```

yyassi-heartex committed:
Update web/libs/editor/src/assets/styles/_mixins.scss

yyassi-heartex commented on web/libs/editor/src/assets/styles/_mixins.scss:
```diff
@@ -1,30 +1,15 @@
     }
-  } @else if $breakpoint == "tablet" {
-    @media (width <= 1200px) {
+  } @else if $breakpoint =="tablet" {
```
```suggestion
  } @else if $breakpoint == "tablet" {
```

yyassi-heartex committed:
Update web/libs/editor/src/assets/styles/_mixins.scss

yyassi-heartex commented on web/libs/editor/src/assets/styles/_mixins.scss:
```diff
@@ -1,30 +1,15 @@
 @mixin respond($breakpoint) {
   @if $breakpoint == "phone" {
-    @media (width <= 760px) {
+    @media (width <=760px) {
```
```suggestion
    @media (width <= 760px) {
```

yyassi-heartex committed:
Update web/libs/editor/src/assets/styles/_mixins.scss

yyassi-heartex commented on web/libs/editor/src/assets/styles/_mixins.scss:
```diff
@@ -4,27 +4,12 @@
     }
   } @else if $breakpoint == "tablet" {
-    @media (width <= 1200px) {
+    @media (width <=1200px) {
```
```suggestion
    @media (width <= 1200px) {
```

yyassi-heartex committed:
Update web/libs/editor/src/assets/styles/_mixins.scss

yyassi-heartex committed:
Merge branch 'develop' into fb-fit-248/scrollbars-white

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-fit-248/scrollbars-white

yyassi-heartex committed:
make sure we give the test a chance to finish shaking out before we check results
